torch/csrc/jit/serialization/export.cpp

```diff
@@ -69,47 +69,37 @@ void validateBlock(
   throw std::runtime_error(                        \
       std::string("ONNX export failed: ") + name + \
       "\n\nGraph we tried to export:\n" + b->owningGraph()->toString());
-    if (node->kind() == prim::PythonOp) {
```

Comment:
Synced offline with David. This part of code, as well as the following part which asserts operator not in prim namespace, should not be removed. Need to look into why certain tests were failing when these are not removed.

Reply:
PR https://github.com/pytorch/pytorch/pull/64944